predictor.js

```diff
@@ -2044,8 +2044,18 @@ const Koviko = {
             newStatisticValue = (state.resources.expectedSS+state.resources.nonDungeonSS) / totalMinutes;
             legend="SS Expected";
           } else if (trackedStat.name=="act") {
-            newStatisticValue= loop / totalMinutes;
-            legend=actions[finalIndex].name;
+            let finalAction = actions[finalIndex];
+            if (finalAction.disabled) {
+              // Find the last non-disabled action
+              for (let i = finalIndex - 1; i >= 0; i--) {
+                if (!actions[i].disabled) {
+                  finalAction = actions[i];
+                  break;
+                }
+              }
+            }
```

Comment:
I think this entire code block may be unnecessary, see lines 1657-1660 where `finalIndex` is initially set: https://github.com/dmchurch/omsi-loops/blob/93f14f4f1c8fa7c43322938403d83a793c835290/predictor.js#L1657-L1660

Check and see if things work the same without it.